pkg/gohai/cpu/cpu_windows.go

```diff
@@ -247,43 +230,41 @@ func GetCpuInfo() (map[string]string, error) {
 		defer k.Close()
 
 		dw, _, err := k.GetIntegerValue("~MHz")
-		if err == nil {
-			cpuInfo["mhz"] = strconv.Itoa(int(dw))
-		}
+		cpuInfo.Mhz = utils.NewValueFrom(float64(dw), err)
```

Comment:
Previously it was int and now is float. I check registry it is DWORD

Reply:
This is not an error: on Linux the CPU frequency is collected as a float (and it is sent as such in the metadata), one of the goals of the new API is that each piece of information has the same format/type no matter the platform (it helps preventing parsing issues for example), having it as a float on Windows too makes the API more uniform (and I think since it's a frequency in Mhz it makes sense to have a float)